packages/vue/src/DataDisplay/FloatContent/OcFloatContent.vue

```diff
@@ -0,0 +1,132 @@
+<script setup>
+import { CopyTooltip, Dropdown, DropdownItem, Icon } from "@/orchidui";
+import { ref } from "vue";
+
+const props = defineProps({
+  modelValue: {
+    type: Boolean,
+    default: false,
+  },
+  isClose: {
+    type: Boolean,
+    default: true,
+  },
+  isDropdown: {
+    type: Boolean,
+    default: true,
+  },
+  title: String,
+  description: String,
+  dropdownOptions: {
+    type: Object,
+    default: () => ({
+      top: [],
+      bottom: [],
+    }),
+  },
+  dropdownProps: {
+    type: Object,
+    default: () => ({}),
+  },
+  position: {
+    type: String,
+    validator: (v) => ["left", "right"].includes(v),
+    default: "right",
+  },
+});
+
+const emit = defineEmits(["update:model-value"]);
+
+const isOpen = ref(false);
+</script>
+
+<template>
+  <div
+    class="fixed top-0 bottom-0 shadow-xl bg-oc-bg-light transition-all h-full z-50 overflow-y-auto"
+    :class="{
+      'right-0': modelValue && position === 'right',
+      'right-[-999px] w-0': !modelValue && position === 'right',
+      'left-0': modelValue && position === 'left',
+      'left-[-999px] w-0': !modelValue && position === 'left',
+    }"
+  >
+    <div v-if="modelValue">
+      <div
+        class="flex border-oc-gray-200 gap-x-9 justify-between p-5 items-start border-b sticky bg-oc-bg-light top-0"
+      >
```

Comment:
Is there a need for an extra `div` or can be as below + removing matching closing `div`
```suggestion
    <div v-if="modelValue" class="flex border-oc-gray-200 gap-x-9 justify-between p-5 items-start border-b sticky bg-oc-bg-light top-0">
```

Reply:
@bdebashi We need because we hide 2 blocks -> header and slot